ansible/roles/test/tasks/buffer_profile.yml

```diff
@@ -0,0 +1,30 @@
+#
+# ansible-playbook test_sonic.yml -i inventory --limit arc-switch1028 --tags buffer_profile -e skip_interface_check=true
+#
+#
+
+- set_fact:
+    port_name="Ethernet64"
+    minigr="[ 'minigraph0.xml', 'minigraph1.xml', 'minigraph2.xml']"
```

Comment:
Can these 3 files be picked up by fileglob or similar mechanism, instead of explicitly setting them?

Reply:
2.1 the test can run on any device but will test msn27xx platform template only. Currently this is the only platform which uses generated buffers config and not static.
2.2 yes, with some updates it can be used for the other platforms

about fileglob: maybe, but in this test the order is important

Reply:
Andriy,

The ordering part is quite tricky. I am not confident that another developer changing it will understand the constraints clearly.

Do you think you can change the minigr list to a dictionary, you can explicitly name the comparison target in the dictionary?

Cheers,
Ying